exchange-common/src/main/scala/com/vesoft/exchange/common/writer/ServerBaseWriter.scala

```diff
@@ -248,10 +345,10 @@ class NebulaGraphClientWriter(dataBaseConfigEntry: DataBaseConfigEntry,
   }
 
   override def writeEdges(edges: Edges, ignoreIndex: Boolean = false): String = {
-    val sentence = toExecuteSentence(config.name, edges, ignoreIndex)
+    // val sentence = toExecuteSentence(config.name, edges, ignoreIndex)
```

Comment:
you can just remove this line. same for writeVertices.

Reply:
OK, I got it~

Reply:
> you can just remove this line. same for writeVertices.

Done~